src/behaviours/oculus-touch-joystick-dpad4.js

```diff
@@ -0,0 +1,31 @@
+import { angleTo4Direction, angleTo8Direction } from "../utils";
+
+// @TODO specify 4 or 8 direction
+function oculus_touch_joystick_dpad4(el, outputPrefix) {
+  this.angleToDirection = angleTo4Direction;
+  this.outputPrefix = outputPrefix;
+  this.centerRadius = 0.6;
+  this.previous = "none";
+  this.hapticIntensity = "low";
+  this.emitDPad4 = this.emitDPad4.bind(this);
+  el.addEventListener("axismove", this.emitDPad4);
+}
+
+oculus_touch_joystick_dpad4.prototype = {
+  emitDPad4: function(event) {
+    const x = event.detail.axis[0];
+    const y = event.detail.axis[1];
+    const inCenter =
```

Comment:
This should be checking `x*x + y*y < centerRadiusSquared`, its currently a center square instead of a circle. centerRadiusSquared should be computed once in the constructor